embedchain/loaders/wikipedia.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import wikipedia
+
+from embedchain.loaders.base_loader import BaseLoader
+
+
+class WikipediaLoader(BaseLoader):
+    def load_data(self, title):
+        """Load data from a wikipedia."""
+        output = []
+
+        try:
+            page = wikipedia.page(title)
+        except wikipedia.exceptions.DisambiguationError as e:
+            logging.error(f"DisambiguationError: {e.options}")
+            page = None
+        except wikipedia.exceptions.PageError as e:
+            logging.error(f"PageError: {e}")
+            page = None
+
+        if page is not None:
+            meta_data = {
+                "url": page.url,
+                "title": page.title
+            }
+            output.append({
+                "content": page.content,
+                "meta_data": meta_data,
+            })
+
+        return output
```

Comment:
I know other loaderes do it this way, but please don't add an empty array and then append to it. You can make the return one step.

Reply:
This seemed wrong to me too. When I am adding a page which has DisambiguationError or PageError, it shows that the document was already added although an empty array was passed.

But I didn't quite get you, I can return one step? Can you elaborate?

Reply:
`return [{...}]`

you don't even need to declare an output variable.